### ISSUE TITLE
[Feature request]Delete the executable automatically after each run.

### ISSUE THREAD

**Issue by tgsheng** [formulahendry/vscode-code-runner#848]
**Is your feature request related to a problem? Please describe.**
Description of what the problem is. Ex. I'm always frustrated when [...]

**Describe the solution you'd like**
Description of what you want to happen.

**Comment by gosoccerboy5** [comment]
I agree - when I run Kotlin, the .jar files clutter up my files. You can customize the actions by editing the extension settings
